Shared/Samples/Show device location with NMEA data sources/FileNMEASentenceReader.swift

```diff
@@ -93,10 +96,6 @@ extension ShowDeviceLocationWithNMEADataSourcesView {
             timer = nil
             messages = nil
         }
-        
-        deinit {
-            stop()
```

Comment:
This isn't needed because when the sample's main view disappears, it runs `reset` method that also stops the sentence generation.